### PR TITLE
Adds script to reformat lad table to give multiple ranges when needed

### PR DESCRIPTION
**Pull request by ErinCain**
If you could do a quick review and merge that would be great!

### CODE REVIEW

File: refomat_lad_table.R [hunk 1]
```diff
@@ -0,0 +1,23 @@
+lad_raw <- read_csv(here::here("analysis", "RiverLAD.csv"))
```
Comment (alvizek) [inline refomat_lad_table.R:1]: can you please add the google cloud pull or source some doc that pulls in this csv.

File: refomat_lad_table.R [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  slice_min(fl_start) |>
+  rename(fl_start_1 = fl_start,
+         fl_end_1 = fl_end) |>
+  left_join(filter(lad_counts, count > 1) |> 
+              slice_max(fl_start) |>
+              rename(fl_start_2 = fl_start,
+                     fl_end_2 = fl_end)) |>
+  select(-count) |> glimpse()
+
+write_csv(all_lad, "analysis/river_lad_fl_and_dates.csv")
```
Comment (alvizek) [inline refomat_lad_table.R:23]: can you please save this R script in the analysis folder as well.

**Review by alvizek — APPROVED** [review]

File: refomat_lad_table.R (hunk 2 above)
Comment (ErinCain) [reply refomat_lad_table.R:23]: moved

File: refomat_lad_table.R (hunk 1 above)
Comment (ErinCain) [reply refomat_lad_table.R]: added google cloud pull

**Pull request merged**